### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by shivupa**
I ran [asottile/pyupgrade](https://github.com/asottile/pyupgrade) on the code base to remove anything we were doing with python 2.
With python 3.6 compatibility: `find . -name "*.py" -exec pyupgrade --py36-plus {} \;`

I understand this might be a lot to look at so I could break it into separate PRs. 

This can go in after we finish some of the more pressing PRs.

### CODE REVIEW

File: cclib/io/moldenwriter.py [hunk 1]
```diff
@@ -171,7 +170,7 @@ def _mo_from_ccdata(self):
             for j in range(len(moenergies[i])):
                 lines.append(' Sym= %s' % syms[i][j])
                 moenergy = utils.convertor(moenergies[i][j], 'eV', 'hartree')
-                lines.append(' Ene= {:10.4f}'.format(moenergy))
+                lines.append(f' Ene= {moenergy:10.4f}')
                 lines.append(' Spin= %s' % spin)
```
Comment (berquist) [inline cclib/io/moldenwriter.py:174]: Stupid tools...

**Comment by berquist** [comment]
Semi-related, http://cclib.github.io/how_to_install.html still shows that we test against Python 2.

**Comment by langner** [comment]
As per discussion, @shivupa offered to break this up to into separate focused changes (super, exceptions, file opens, etc.).

**Comment by shivupa** [comment]
Following the merge of #1121, I will close this. The changes have been implemented in #1099 and #1121

**Pull request closed without merging**